ginga/rv/plugins/Pick.py

```diff
@@ -1653,12 +1752,14 @@ def redo_quick(self):
         self.cuts_plot.plot(xpts, xarr, color=hl.color,
                             xtitle="Line Index", ytitle="Pixel Value",
                             title=None, rtitle="Cuts",
-                            alpha=1.0, linewidth=1.0, linestyle='-')
+                            alpha=1.0, linewidth=1.0, linestyle='-',
+                            marker='x', label='X')
```

Comment:
@ejeschke , while update the doc, I decided also to update the look of Cuts plot. But if you disagree, I can back this out, please let me know. Thanks!

Reply:
@pllim, no problem. Actually, I have a PR that updates the Cuts plot in a substantial way for better performance. I will post it shortly after rebasing on top of this, assuming it is merged.

Reply:
Okay, sorry for the potential conflicts. Just waiting for @obi-wan76 or @shanosborne to approve.